### ISSUE TITLE
Remove ResultModelManager remote from DESCRIPTION

### ISSUE THREAD

**Issue by egillax** [OHDSI/PatientLevelPrediction#471]
Now that `ResultModelManager` is in CRAN we need to remove the github remote from DESCRIPTION

**Comment by jreps** [comment]
We can also use OhdsiShinyAppBuilder from CRAN instead of ShinyAppBuilder

**Comment by egillax** [comment]
Yes I've started work on all the dependencies management for the new release in https://github.com/OHDSI/PatientLevelPrediction/tree/dependancies 

It's at least passing tests for now so should be close to being finished.
